### PR TITLE
Corrected the uploading bug for CSV file

### PR DESCRIPTION
**Pull request by ourcach**

### CODE REVIEW

File: sendgrid_backend/mail.py [hunk 1]
```diff
@@ -145,7 +145,9 @@ def _build_sg_mail(self, msg):
 
                 attachment.filename = filename
                 # todo: Read content if stream?
-                attachment.content = base64.b64encode(content)
+                if isinstance(content, str):
+                    content = content.encode()
+                attachment.content = base64.b64encode(content).decode()
```
Comment (sklarsa) [inline sendgrid_backend/mail.py]: Why the additional `.decode()`?

Comment (ourcach) [reply sendgrid_backend/mail.py]: Because if the attachment content is not a str, the backend will fail when sending the email

Comment (sklarsa) [reply sendgrid_backend/mail.py]: Cool thanks! Does this work against the sendgrid API in both python 2 and 3?

Comment (ourcach) [reply sendgrid_backend/mail.py]: Yes, I ran the tests in python 2 and 3.

Comment (sklarsa) [reply sendgrid_backend/mail.py]: Thanks!!

**Pull request merged**